src/Flash.tsx

```diff
@@ -45,6 +45,8 @@ export interface Props {
    * Value to display. The only required prop.
    */
   value: number;
+
```

Comment:
Let's add a JSDoc comment here too, because these comments get parsed and show up in the Storybook interface as documentation.

Reply:
Let's update the README too, as all of our available props are documented there as well.